native/src/components/CategoryListItem.tsx

```diff
@@ -82,15 +83,19 @@ const CategoryListItem = ({
         </CategoryEntryContainer>
       </DirectionContainer>
     </FlexStyledLink>
-    {subCategories.map(subCategory => (
-      <SubCategoryListItem
-        key={subCategory.path}
-        subCategory={subCategory}
-        resourceCache={resourceCache[subCategory.path] ?? {}}
-        onItemPress={onItemPress}
-        language={language}
-      />
-    ))}
+    <List
```

Comment:
Maybe you can rename this file and component to `CategoryList.tsx`, as this component does not render one listItem, as the name says, but the whole list. This was the case before your changes, but maybe you can nevertheless adjust it.

Reply:
@ztefanie I think this is not correct. This renders one list item which, in turn, includes a list of sub category items. Therefore the name should stay the way it is.